packages/components/src/Panel/Panel.test.tsx

```diff
@@ -0,0 +1,170 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2020 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import '@testing-library/jest-dom/extend-expect'
+import { fireEvent } from '@testing-library/react'
+import 'jest-styled-components'
+import React, { useState } from 'react'
+import { renderWithTheme } from '@looker/components-test-utils'
+import { Icon } from '../Icon'
+import { MenuGroup, MenuItem, MenuList } from '../Menu'
+import { Panel, Panels, usePanel } from './'
+
+describe('Panel', () => {
+  const UsePanelHook = () => {
+    const [isOpen, setOpen] = useState(false)
+    const open = () => setOpen(true)
+    const canClose = () => true
+
+    const { panel } = usePanel({
+      canClose,
+      content: 'Panel content',
+      direction: 'left',
+      isOpen,
+      setOpen,
+      title: 'Panel Hook',
+    })
+
+    return (
+      <>
+        {panel}
+        <MenuList>
+          <MenuGroup>
+            <MenuItem onClick={open} icon="Check">
+              Option A
+            </MenuItem>
+            <MenuItem icon="Check">Option B</MenuItem>
+          </MenuGroup>
+        </MenuList>
```

Comment:
Simplify composition here to not have cross-component test

```suggestion
        <ul>
            <li><button onClick={open} icon="Check">Option A</button></li>
            <li icon="Check">Option B</li>
        </ul>
```